Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -1346,11 +1336,12 @@ void Renderer::SwapImpl(u32 xfbAddr, u32 fbWidth, u32 fbStride, u32 fbHeight,
   std::swap(flipped_trc.top, flipped_trc.bottom);
 
   // Copy the framebuffer to screen.
-  DrawFrame(0, flipped_trc, rc, xfbAddr, xfbSourceList, xfbCount, fbWidth, fbStride, fbHeight);
+  glBindFramebuffer(GL_FRAMEBUFFER, 0);
+  BlitScreen(sourceRc, flipped_trc, xfb_texture->GetRawTexIdentifier(), xfb_texture->GetConfig().width, xfb_texture->GetConfig().height);
 
   // The FlushFrameDump call here is necessary even after frame dumping is stopped.
   // If left out, screenshots are "one frame" behind, as an extra frame is dumped and buffered.
-  FlushFrameDump();
+  /*FlushFrameDump();
```

Comment:
I'm planning on going through and removing a ton of code, the commented out code was just in case I needed to quickly reference it for anything.  I haven't done it yet, just to avoid too many merge conflicts.  Don't worry, I hate it when people leave commented out code in commits!